pydax/_high_level.py

```diff
@@ -51,19 +51,21 @@ def init(update_only: bool = True, **kwargs: Any) -> None:
         the current working directory, but will be converted to the absolute path immediately in this function.
         Defaults to: ~/.pydax/data
     """
-    global _global_config
+    global _global_config, _schemata
 
     if update_only:
         # We don't use dataclasses.replace here because it is uncertain whether it would work well with
         # pydantic.dataclasses.
         prev = dataclasses.asdict(_global_config)  # type: ignore [name-defined]
         prev.update(kwargs)
         _global_config = Config(**prev)  # type: ignore [name-defined]
+        for config in kwargs.keys():
+            if 'SCHEMA' in config:
+                _schemata = None
+                break
     else:
         _global_config = Config(**kwargs)  # type: ignore [name-defined]
-        # TODO: Should update _schemata to None here. This can't be done right now because there's a circular imports.
-        # We should address this by moving all high-level APIs to one module and they should not be used by our
-        # low-level functions.
+        _schemata = None
```

Comment:
@xuhdev do you think it would be better to call `load_schemata` directly from `init` so that when a user inits the configs, the schemata are loaded and cached?

Reply:
Currently the code uses a lazy mode, because `init` is automatically called upon the package is loaded.  Otherwise this may cause significant lag in a poor Internet connection environment when a package simply runs `import pydax`.

Reply:
gotcha, makes sense